plugin/src/main/kotlin/org/neotech/plugin/rootcoverage/JaCoCoConfiguration.kt

```diff
@@ -76,6 +79,11 @@ internal fun Project.getExecutionDataFileTree(includeUnitTestResults: Boolean, i
         buildFolderPatterns.add("outputs/managed_device_code_coverage/*/*/coverage.ec")
         // In case of flavors coverage is nested an additional 2 folder deeper
         buildFolderPatterns.add("outputs/managed_device_code_coverage/*/flavors/*/*/coverage.ec")
```

Comment:
I think we could remove this pattern (and the one above it), since the new patterns will match these anyways.

Reply:
Good catch! Removed them.